wifi/scan.py

```diff
@@ -154,3 +157,7 @@ def normalize(cell_block):
         cell.encryption_type = 'wep'
 
     return cell
+
+# all with sudo arg does not work
+# Everything is ok when launched in admin mode
+# Seems to depend on distribution, worked on xubuntu...
```

Comment:
I use Debian 7 and I concluded the computer was waiting for me to enter a password. Read more on : http://doc.ubuntu-fr.org/sudoers .